### ISSUE TITLE
Possible to use TLBE for a replay?

### ISSUE THREAD

**Issue by fortysixandtwo** [veger/TLBE#24]
Factorio does have the "enable replay" tickbox when starting a new game which will supposedly save everything that ever happened inside the savegame.

Now I wonder if it would be possible to use/setup TLBE while watching a replay.
When watching a replay I noticed that everything seems to happen exactly as it did in the original game
including which menus were opened at which time (inventory, production menus, etc).

Any idea if this would be possible at all?
I see two problems: 
- Does the replay even load if you didn't play that savegame with TLBE enabled?
- If all the menu interactions are also replayed, how can you possibly setup TLBE?

It would be pretty cool if it were possible as you could easily make some taxing recordings (high resolution, multiple cameras, higher framerate) while running the replay in offline mode - possibly scripted?

**Comment by veger** [comment]
> Does the replay even load if you didn't play that savegame with TLBE enabled?

You cannot add new mods (or any other modification) on a replay, so even if TLBE would have been loaded, you cannot change camera settings (as the game is not able to figure out that this does not influence anything on the game play, has to do with consistency/predictability that is used by Factorio (multi player))

> If all the menu interactions are also replayed, how can you possibly setup TLBE?

Exactly... :disappointed:

There are people who [modify the `control.lua` of a game](https://www.reddit.com/r/factorio/comments/alwj33/is_it_possible_to_take_daytimeonly_screenshots/) to take screenshot(s) afterwards, one of the devs even has a [gist showing how it could be done](https://gist.github.com/Bilka2/579ec217ec38e055328e4a23f2fd71a3).

Ofcourse these are pretty simple screenshots compared to what TBLE does... Maybe this could be used to (ab)use the mod to start taking screenshots? (Assuming the mod was loaded in the game already) Or copy&paste all mod code into the `control.lua`...?

**TL;DR** I am pretty sure the mod itself cannot be used (as a mod), but hacking `control.lua` (theoretically) would be able to.
